scripts/cut_release.sh

```diff
@@ -131,29 +131,26 @@ push_to_npm() {
 }
 
 build_examples() {
-    printf "${blue}Building styleguide...${end}"
-    yarn build:prod:examples || return 1
-    printf "${green}Built styleguide!${end}"
     printf "${blue}Building storybook...${end}"
     yarn build:prod:storybook || return 1
     printf "${green}Built storybook!${end}"
 }
 
 push_to_gh_pages() {
-    printf "${blue}Pushing styleguide to gh-pages...${end}"
+    printf "${blue}Pushing storybook to gh-pages...${end}"
     if [[ $(git branch | grep -w "gh-pages") != "" ]] ; then
         git branch -D gh-pages || return 1
         printf "${green}Deleted existing gh-pages branch!${end}"
     fi
     git checkout -b gh-pages || return 1
     rm -rf build
-    cp -R styleguide/. ./ || return 1
+    cp -R storybook/. ./ || return 1
     cp examples/gitignore .gitignore || return 1
     git rm -rf --cached . || return 1
     git add -A || return 1
     git commit --no-verify -am "build(examples): v$VERSION" || return 1
```

Comment:
```suggestion
    git commit --no-verify -am "build(storybook): v$VERSION" || return 1
```
I think its okay to update this too